runtime/redirector/module.xml

```diff
@@ -23,17 +23,6 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 <module xmlns:xi="http://www.w3.org/2001/XInclude">
 	<xi:include href="../j9vm/j9vmnatives.xml"><xi:fallback/></xi:include>
 
-	<exports group="jdk11">
-		<export name="_JVM_GetCallerClass@4"/>
```

Comment:
I note the j9vmnatives.xml doesn't include this export.

Reply:
`JVM_GetCallerClass` is included in the `j9vmnatives` group (its signature changed in Java 11):

```
	<exports group="j9vmnatives">
		<export name="_JVM_GetCallerClass@8">
			<exclude-if condition="spec.java11"/>
		</export>
		<export name="_JVM_GetCallerClass@4">
			<include-if condition="spec.java11"/>
		</export>
```